smacc2_ci/docker/ros_rollingAndGalactic_ubuntu_20.04/buildGalactic.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd ../../../..
+echo "Building DOCKER from directory `pwd`"
+
+#TODO: Implement user selection of ROS2 Distro
+#Ex sudo docker build --build-arg ROS2_DISTRO=galactic
+
+sudo docker build -f $DIR/Dockerfile -t smacc2_galactic_ubuntu_2004 .
```

Comment:
I am not sure if sudo here is really neded. Are you in `docker` group (or some similar name?

```suggestion
docker build -f $DIR/Dockerfile -t smacc2_galactic_ubuntu_2004 .
```

Reply:
Found that sudo is necessary at least in my case.
![Screenshot from 2021-09-20 12-25-49](https://user-images.githubusercontent.com/44791484/134062404-e658cedc-13ab-4cc5-ad9e-2eb6d6fdf6a3.png)

Reply:
It don't think I have a 'docker' group attached to my user.  Perhaps that is why I needed sudo.
![Screenshot from 2021-09-20 12-32-45](https://user-images.githubusercontent.com/44791484/134063271-74084ad7-92d5-4091-b33b-7b5a32a5e4f3.png)